system-test/cluster.ts

```diff
@@ -0,0 +1,326 @@
+          },
+        },
+      });
+    } else {
+      assert.equal(isConfigDefined, false);
+    }
+  }
+
+  async function createNewInstance(clusters: ClusterInfo[]): Promise<void> {
+    const instanceId: string = generateId('instance');
+    instance = bigtable.instance(instanceId);
+    const [, operation] = await instance.create({
+      clusters,
+      labels: {
+        time_created: Date.now(),
+      },
+    });
+    await operation.promise();
+  }
+  async function createStandardNewInstance(
+    clusterId: string,
+    nodes: number
+  ): Promise<void> {
+    return await createNewInstance(standardCreationClusters(clusterId, nodes));
+  }
+  function standardCreationClusters(
+    clusterId: string,
+    nodes: number
+  ): ClusterInfo[] {
+    return [
+      {
+        id: clusterId,
+        location: 'us-east1-c',
+        nodes,
+      },
+    ];
+  }
+  afterEach(async () => {
+    await instance.delete();
+  });
+  describe('Create cluster', () => {
+    describe('With manual scaling', () => {
+      let clusterId: string;
+      let cluster: Cluster;
+      beforeEach(async () => {
+        clusterId = generateId('cluster');
+        await createStandardNewInstance(clusterId, 2);
+        cluster = instance.cluster(clusterId);
+      });
+      it('should create an instance with clusters for manual scaling', async () => {
+        await checkMetadata(cluster, {nodes: 2}, false);
+      });
+      it('should create an instance and then create a cluster for manual scaling', async () => {
+        const clusterId2: string = generateId('cluster');
+        const cluster2 = instance.cluster(clusterId2);
+        await cluster2.create({
+          location: 'us-west1-c',
+          nodes: 3,
+        });
+        await checkMetadata(cluster2, {nodes: 3}, false);
+      });
+      describe('Using an incorrect configuration', () => {
+        let cluster2: Cluster;
+        beforeEach(async () => {
+          const clusterId2: string = generateId('cluster');
+          cluster2 = instance.cluster(clusterId2);
+        });
+        it('should throw an error when providing no cluster configuration', async () => {
+          try {
+            await cluster2.create({
+              location: 'us-west1-c',
+            });
+            assert.fail();
+          } catch (e) {
+            assert.equal(e.message, ClusterUtils.noConfigError);
+          }
+        });
+        it('should throw an error when providing manual and autoscaling configurations', async () => {
+          try {
+            await cluster2.create({
+              location: 'us-west1-c',
+              nodes: 2,
+              minServeNodes: 3,
+            });
+            assert.fail();
+          } catch (e) {
+            assert.equal(e.message, ClusterUtils.allConfigError);
+          }
+        });
+        it('should throw an error providing all autoscaling configurations', async () => {
```

Comment:
should this say "should throw an error when missing some autoscaling configurations"?